### PR TITLE
exception docu, close #963

### LINKED ISSUE #963: document all exceptions plus cause example
* [x]	BadGatewayException
* [x]	BadRequestException
* [x]	ClientErrorException
* [x]	ConcurrentModificationException
* [x]	DeprecationExceptionSphereClientDecorator
* [ ]	ErrorResponseException
* [x]	ExceptionDocumentation
* [x]	GatewayTimeoutException
* [x]	HttpException
* [x]	InternalServerErrorException
* [x]	InvalidClientCredentialsException
* [x]	InvalidRangeException
* [x]	InvalidTokenException
* [x]	InvertedBoundsException
* [x]	JsonException
* [x]	NotFoundException
* [x]	RequestEntityTooLargeException
* [x]	SameExclusiveBoundsException
* [x]	ServerErrorException
* [x]	ServiceUnavailableException
* [x]	SphereDeprecationException
* [x]	SphereException
* [x]	SphereServiceException
* [x]	SphereTimeoutException
* [x]	UnauthorizedException

### PR DESCRIPTION
**Pull request by schleichardt**
@lauraluiz please review

### CODE REVIEW

File: commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java [hunk 1]
```diff
@@ -1,5 +1,10 @@
 package io.sphere.sdk.client;
 
+/**
+ * Unauthorized access to the commercetools platform wither through invalid client credentials or tokens.
```
Comment (lauraluiz) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java]: wither?

**Comment by lauraluiz** [comment]
Great docu! :+1:

File: commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java (hunk 1 above)
Comment (schleichardt) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java]: I found the term here: http://immutables.github.io/immutable.html#copy-methods

Comment (lauraluiz) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java]: Ah nice. Although it's hard to read if you don't know it. Maybe between quotes? "wither"

Comment (schleichardt) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java]: Oops, it should be "Unauthorized access to the commercetools platform with invalid client credentials or tokens."

Comment (lauraluiz) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java]: Ah that makes sense :D

Comment (schleichardt) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java]: I think it was supposed to be with either that merged to wither.

**Pull request merged**